CMakePresets.json

```diff
@@ -78,7 +77,7 @@
         {
             "name": "docs",
             "configurePreset": "docs",
-            "targets": ["Doxygen","Sphinx","Antora"]
+            "targets": ["docs","Doxygen","Sphinx","Antora"]
```

Comment:
I doesn't understand why "docs" added ?

Reply:
docs is a new target that does all other 3